### DISCUSSION TITLE
`Project does not have access to the requested model.`

### DISCUSSION THREAD

**Discussion by gtzo** [deepgram/community#289] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

I'm trying to stream a transcription and getting a project access denied error.
`Project does not have access to the requested model. 2a334700-9e13-4e32-a633-67dd8fc90c1a`

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

wss://api.deepgram.com/v1/listen?model=nova?punctuate=true&encoding=linear16&sample_rate=16000

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

2a334700-9e13-4e32-a633-67dd8fc90c1a

**Comment by jkroll-deepgram** [comment]
Hi @gtzogana, sorry that you have been having trouble accessing Nova. If you look closely at the Deepgram URL, notice that there is an extra question mark after `model=nova` instead of an ampersand. That results in the model's name being interpreted as `nova?punctuate=true`, which does not exist, and is raising this misleading error that you don't have access to the model.

Can you please try switching the `?` to `&`, so the full URL is: `wss://api.deepgram.com/v1/listen?model=nova&punctuate=true&encoding=linear16&sample_rate=16000`

**Reply by gtzo** [reply]
oof. thanks!
